docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,253 +1,396 @@
+--- 
+title: Getting Started With Role-Based Access Control
+description: Teleport Role-Based Access Control.
+tocDepth: 3
 ---
-title: Getting Started With Access Controls
-description: Get started using Access Controls.
----
-
-In Teleport, any local, SSO, or robot user can be assigned one or several roles.
-Roles govern access to databases, SSH servers, Kubernetes clusters, Windows
-desktops, and web apps.
 
-We will start with local users and preset roles, assign roles to SSO users, and
-wrap up with creating your own role.
+Teleport RBAC (Role-Based Access Control) is a system for managing who can
+access what within your infrastructure. Instead of assigning permissions
+directly to users, you create roles that define sets of permissions, then assign
+those roles to users.
+
+This guide helps you understand and implement Role-Based Access Control (RBAC)
```

Comment:
I think the most common thing is to spell out the acronym on first use, and then use the acronym going forward.

I would update line 7 tog spell out RBAC before using the acronym, and then use only the acronym here.